assembl/static2/js/app/utils/translations.js

```diff
@@ -302,7 +302,7 @@ const Translations = {
       menu: {
         phase: "Phase %{count} - %{description}",
         language: "Editer la langue du débat",
-        legalNoticeAndTerms: "CGU & mentions légales"
+        legalNoticeAndTerms: "CGU & Mentions Légales"
```

Comment:
I saw this in the mock-up but in French we don't use capital letters for each word in titles (same above for CGU). Check with Fred?

Reply:
done